### DISCUSSION TITLE
How to update Flood UI to latest version?

### DISCUSSION THREAD

**Discussion by gregmichael** [haugene/docker-transmission-openvpn#2245] [Q&A]
Title - how can we update Flood UI which is prefaced to latest version with new features/fixes?

**Comment by notoriousjere** [comment]
You can get it by cloning this repo and then rebuilding the docker image. The flood UI is fairly out of date since the last release was from September 2021. The UI is embedded in the image and not pulled on container creation. See discussions here:

https://github.com/johman10/flood-for-transmission/issues/405#issuecomment-1102856988
https://github.com/haugene/docker-transmission-openvpn/discussions/1921

**Comment by notoriousjere** [comment]
I rebuild the image, which pulled the new flood UI:
https://mega.nz/file/lBAjyBwZ#2CtoPFyq43IyTKIk6FusXNbJ544a10UrHJuBS9ElHhg
